tests/e2e/specs/shopper/shopper-wc-blocks-checkout-purchase.spec.js

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
I'd add a note that it shouldn't be run in a `-dev` mode as a source of truth, just an idea how it works.
Due to reasons you've described.

Reply:
I can do that. 👍